sky/shell/ui/engine.cc

```diff
@@ -262,5 +262,14 @@ void Engine::RequestNavigateHistory(int32_t delta) {
   NOTIMPLEMENTED();
 }
 
+void Engine::StartDartTracing() {
+  sky_view_->StartDartTracing();
+}
+
+void Engine::StopDartTracing(mojo::ScopedDataPipeProducerHandle dataProducer) {
+
```

Comment:
You've got an extra blank line here.

Also, s/dataProducer/producer/ everywhere...